Readme/automation.md

```diff
@@ -0,0 +1,155 @@
+# Running Automation test with Kobiton with TeamCity
+## Table of contents
++ [A. Preparation](#a-preparation)
+  + [1. Prerequisites](#1-prerequisites)
+  + [2. Gettting Kobiton username and API key](#2-gettting-kobiton-username-and-api-key)
+  + [3. Samples](#3-samples)
++ [B. Setup](#b-setup)
+  + [1. Getting desired capabilities](#1-getting-desired-capabilities)
+  + [2. Configuring automation test script](#2-configuring-automation-test-script)
+  + [3. Configuring HockeyApp Webhooks listenner](#3-configuring-hockeyapp-webhooks-listenner)
+  + [4. Configuring TeamCity project](#4-configuring-teamcity-project)
++ [C. Execution](#c-execution)
+  + [1. Runnig automation test on Kobiton devices](#1-running-automation-test-on-kobiton-devices)
+  + [2. Fetch test session data through Kobiton REST API](#2-fetch-test-session-data-through-kobiton-rest-api)
++ [D. Feedback](#d-feedback)
+## A. PREPARATION
+### 1. Prerequisites
++ A server for receiving HockeyApp webhooks with Ruby 2.x installed.
+  > If you haven't installed Ruby on your server, please refer to [Ruby Installation documentation](https://www.ruby-lang.org/en/documentation/installation/) on how to install Ruby on corresponding operation system.
++ Install nodejs: Nodejs is required to run our sample automation test below.
+  > Follow the [installation document](https://nodejs.org/en/download/). 
+### 2. Gettting Kobiton username and API key
+In order to execute test on Kobiton, these two must-have parameters must be provided: Username and API key. They are necessary for authenticating with Kobiton.
+
+> If you don't have a Kobiton account, visit https://portal.kobiton.com/register to create an account.
+
+Follow the instruction at `IV. Configure Test Script for Kobiton` section on [our blog article](https://kobiton.com/blog/tutorial/parallel-testing-selenium-webdriver/) to get Kobiton username and API key.
+### 3. Samples
+In this repository, we have already provided samples for executing automation test with Kobiton through TeamCity:
+  -  [automation-test.js](../sample/automation-script/automation-test.js): script written in NodeJS for execute automation test on Kobiton devices.
+  -  [server.rb](../sample/server/server.rb): script for starting WebHook server.
+
+To use the provided sample, follow these steps:
+1. Fork this repository https://github.com/kobiton/TeamCity-HockeyApp-Appium-ReactNative.
+2. In your TeamCity project, create a `Build configurations` to use the forked repository. If you don't know how to do it, please follow [TeamCity document](https://confluence.jetbrains.com/display/TCD18/Build+Configuration+Template) for instructions.
```

Comment:
a `Build configurations` ?
(If you don't know how to do it) : can be removed [optional], just need 'refer to ... for instructions'.